src/components/TodoContainer.js

```diff
@@ -75,32 +74,59 @@ export default function TodoContainer() {
         id: jsonResponse.id,
       };
 
-      console.log(airtableTodo);
       setTodoList([...todoList, airtableTodo]);
     } catch (error) {
       console.log(error.message);
     }
   };
-  const removeTodo = (id) => {
-    const removeItem = todoList.filter((todo) => todo.id !== id);
 
-    setTodoList(removeItem);
+  const removeTodo = async (id) => {
+    try {
+      const airtableData = {
+        fields: {
+          id: id,
+        },
+      };
+      const options = {
+        method: "DELETE",
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
+        },
+        body: JSON.stringify(airtableData),
+      };
+
+      const url = `
+https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/${id}`;
+
+      const response = await fetch(url, options);
+
+      setTodoList(todoList.filter((todo) => todo.id !== id));
+    } catch (error) {
+      console.log(error.message);
+    }
+  };
+
+  const handleToggleSortDirection = () => {
+    setIsSortAscending(!isSortedAscending);
   };
 
+  const sortedTodos = [...todoList].sort((objectA, objectB) => {
+    const titleA = objectA.title ?? "";
+    const titleB = objectB.title ?? "";
+    if (isSortedAscending) {
+      return titleA.localeCompare(titleB);
+    } else return titleB.localeCompare(titleA);
+  });
```

Comment:
I'm able to break your current sorting by adding extra spaces before my title starting, since a space sorts before `a`. 

You can fix this by performing a `String.trim()` on every new todo item before submitting that value to Airtable